pkg/check/interface.go

```diff
@@ -62,14 +59,18 @@ type KubeClient interface {
 }
 
 type CheckContext struct {
-	Context     context.Context
-	VMConfig    *vsphere.VSphereConfig
-	VMClient    *vim25.Client
-	TagManager  *vapitags.Manager
-	Username    string
-	AuthManager AuthManager
-	KubeClient  KubeClient
-	ClusterInfo *util.ClusterInfo
+	Cache         cache.VSphereCache
+	Context       context.Context
+	VMConfig      *vsphere.VSphereConfig
+	GovmomiClient *govmomi.Client
+	VMClient      *vim25.Client
+	TagManager    *vapitags.Manager
+	Username      string
+	AuthManager   AuthManager
+	KubeClient    KubeClient
+	ClusterInfo   *util.ClusterInfo
+	//Infra            *ocpv1.Infrastructure
```

Comment:
is that not needed?

Reply:
This was there in 4.14.  I was leaving in to keep them in sync.